### DISCUSSION TITLE
Where is data from the web app being stored

### DISCUSSION THREAD

**Discussion by DanRothwell** [ukrobotics/DispenseLibDotNet#16] [Q&A] [answered]
(asking here because the web app doesn't have a feedback option)
When data is saved/loaded rather than exported/imported, where does it go? Is it held locally or on the cloud?
- Protocols
- Plate types
- Calibrations

**Comment by mcounsell** [comment, marked as answer]
Protocols and Calibrations are stored in the cloud

Plate types are pulled from our public plate library here:  https://labware.ukrobotics.app/
